### PR TITLE
SN-60 BUG FIX: Created a weight_files folder and added comments and did somme dead code Refactoring 

### PR DESCRIPTION
**Pull request by deeppatel-11**

### CODE REVIEW

File: Pet_Detection/Pet_Detection.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
+import math
```
Comment (deeppatel-11) [inline Pet_Detection/Pet_Detection.py:1]: Thanks for adding this i deleted it bymistake

**Review by abirshah — APPROVED** [review]
Everything looks good. This should solve the problem.

File: Pet_Detection/Pet_Detection.py [hunk 2]
```diff
@@ -98,7 +98,7 @@ def main():
     mail_bird_classes = getClasses('names_files/mail-bird.names')
 
     # Reading the image you are testing
-    my_img = cv2.imread('test_images/dog.jpg')
+    my_img = cv2.imread('test_images/cat.jpg')
     my_img = cv2.resize(my_img, (800, 600))
```
Comment (abirshah) [inline Pet_Detection/Pet_Detection.py:102]: Thaks for testing with different images.

File: Pet_Detection/Pet_Detection.py [hunk 3]
```diff
@@ -144,7 +150,6 @@ def main():
                     print('Bird found in the pets mouth')
 
         if str(classes[class_ids[i]]) == 'cat' or str(classes[class_ids[i]]) == 'dog':
-
             # This stores the sze of each bounding box into a dictionary
             addingSizeOfBoundingBoxes(bounding_box_size, str(classes[class_ids[i]]), w * h)
```
Comment (abirshah) [inline Pet_Detection/Pet_Detection.py:160]: Good work for removing whitespace. We can refactor more by removing some more whitespace. In the future, development phase.

**Pull request merged**